src/components/ReportActionItem/MoneyRequestReportPreview/MoneyRequestReportPreviewContent.tsx

```diff
@@ -755,21 +765,38 @@ function MoneyRequestReportPreviewContent({
-                                        <View style={[styles.flexRow, styles.alignSelfCenter, styles.gap2]}>
-                                            {carouselTransactions.map((item, index) => (
-                                                <PressableWithFeedback
-                                                    accessibilityRole="button"
-                                                    accessible
-                                                    accessibilityLabel="button"
-                                                    style={[styles.reportPreviewCarouselDots, {backgroundColor: index === currentIndex ? theme.icon : theme.buttonDefaultBG}]}
-                                                    onPress={() => handleChange(index)}
-                                                />
-                                            ))}
+                                    <View style={[styles.expenseAndReportPreviewTextContainer, styles.overflowHidden]}>
+                                        <View
+                                            style={[
+                                                shouldUseNarrowLayout ? styles.flexColumnReverse : styles.flexRow,
+                                                shouldUseNarrowLayout ? styles.alignItemsStretch : styles.alignItemsBaseline,
+                                                styles.justifyContentBetween,
+                                                styles.gap3,
+                                                StyleUtils.getMinimumHeight(variables.h28),
+                                            ]}
+                                        >
+                                            {/* height is needed to avoid flickering on animation */}
+                                            {!shouldShowEmptyPlaceholder && (
+                                                <View style={[buttonMaxWidth, styles.flex1, {height: variables.h40}]}>{reportPreviewActions[reportPreviewAction]}</View>
+                                            )}
+                                            {transactions.length > 1 && (
+                                                <View style={[styles.flexRow, shouldUseNarrowLayout ? styles.justifyContentBetween : styles.gap2, styles.alignItemsBaseline]}>
+                                                    <Text
+                                                        style={[styles.textLabelSupporting]}
+                                                        numberOfLines={1}
+                                                    >
+                                                        Total
+                                                    </Text>
+                                                    <Text
+                                                        style={[styles.headerText]}
+                                                        testID="MoneyRequestReportPreview-currencyAndAmount"
```

Comment:
are you writing any tests that use this specific `testID`? 

If this prop is not currently being used anywhere then please remove it.